src/main/java/peoplesoft/ui/PageSwitcher.java

```diff
@@ -0,0 +1,41 @@
+package peoplesoft.ui;
+
+import peoplesoft.ui.regions.SideBar;
+
+/**
+ * A relation class between mainwindow and sidebar which handles the page switching
+ * todo: to be completed after making PeopleCell and JobCell for the 2 lists and displaying them properly.
+ *
+ * responsible for changing sidebar's colours and listening to commands from mainwindow and switching in main window
+ */
+public class PageSwitcher {
+    private MainWindow mw;
+    private SideBar sb;
+
+    private enum PageValues {
+        OVERVIEW, HELP
+    }
+
+    private void switchOnCommand(PageValues p) throws IllegalArgumentException {
+        assert p != null;
```

Comment:
hmm i guess this will only cause the `IllegalArgumentException` to be thrown earlier, right? but can't hurt to have the assert in either anyway

Reply:
did it for the 'use assert in code' tp progress haha